server/scripts/import-tests/index.js

```diff
@@ -251,6 +253,38 @@ const readDirectoryGitInfo = directoryPath => {
     return { gitSha, gitMessage, gitCommitDate };
 };
 
+const importHarness = () => {
+    const sourceFolder = path.resolve(`${testsDirectory}/resources`);
+    const targetFolder = path.resolve('../', 'client/resources');
+    console.info('Clearing harness directory ...');
+    fse.rmSync(targetFolder, { recursive: true, force: true });
+
+    // Copy source folder
+    console.info('Importing harness ...');
+    fse.copySync(sourceFolder, targetFolder, { filter: (src) => {
+            if (fse.lstatSync(src).isDirectory()) {
+                return true;
+            }
+            if (!src.includes('.html')) {
+                return true;
+            }
+        },
+    });
+
+    // Copy files
+    const commandsJson = 'commands.json';
+    const supportJson = 'support.json';
+    fse.copyFileSync(
+        `${testsDirectory}/${commandsJson}`,
+        `${targetFolder}/${commandsJson}`
+    );
```

Comment:
This file MAY exist if a much older commit is used to run the import script

Reply:
I put in a check to see if that file exists.